tools/patchwork_integration_test.sh

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+set -e
+HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PUBIN_PATH=$1
+SINCE=$2
+MBOX=test-mbox.mbox
+PROJECT=test-patchwork
+INITIAL_IMPORT_SIZE=50 # Percentage of mbox to be imported as a raw mbox
+INITIAL_IMPORT_MBOX=initial.mbox
+
+clean_up() {
+	1>/dev/null 2>&1 docker stop pasta-patchwork
+}
+
+trap clean_up EXIT
+
+parse_pubin() {
+	git -C $PUBIN_PATH --no-pager log --pretty=format:%H --since=$SINCE --reverse | \
+		xargs -n 1 -I {} bash -c "
+				git -C $PUBIN_PATH --no-pager show --no-patch --pretty=format:'From %H %ad' {};
+				echo;
+				git -C $PUBIN_PATH --no-pager show {}:m;
+				echo -en '\n\n';
+			"
+}
+
+get_config() {
+	if [ $1 = "mbox" ]
+	then
+		initial_archive=$INITIAL_IMPORT_MBOX
+	else
+		initial_archive=""
+	fi
+	config=$(cat <<-END
+		[PaStA]
+		MODE = "mbox"
+		UPSTREAM = "HEAD^..HEAD"
+
+		[mbox]
+		[mbox.patchwork]
+		url='http://$container_addr:8000/api/'
+		projects = [{id=$project_id, initial_archive="$initial_archive", list_email="test@test.com"}]
```

Comment:
test@test.com is not the valid email address. Please pass the correct list email address via commandline.

Reply:
That address doesn't really play a role in the analyses. I think we dont need to pass it from the command line

Reply:
Still, test@test.com is incorrect.

Reply:
Umm, so do we take the list address as an argument from the user? That seems redundant

Reply:
I'd suppose that. why would that be redundant? On patchwork-side we also have to specify a project name AND a list email address, which seems kind of redundant as well.

Reply:
Ok, added another argument.

Reply:
It's still test@test.com